rllib/algorithms/appo/tf/appo_tf_rl_module.py

```diff
@@ -49,8 +46,10 @@ def output_specs_train(self) -> List[str]:
     @override(PPOTfRLModule)
     def _forward_train(self, batch: NestedDict):
         outs = super()._forward_train(batch)
-        old_pi_inputs_encoded = self.old_encoder(batch)[ENCODER_OUT][ACTOR]
-        old_action_dist_logits = self.old_pi(old_pi_inputs_encoded)
+        old_pi_inputs_encoded = tf.stop_gradient(
```

Comment:
Added the `stop_gradient's`, just in case.
Note that for the torch version, adding `.detach()` here will create a weird error due to DDP, not handling this well. But it works fine on the torch side either way.